exs_numpy.py

```diff
@@ -0,0 +1,423 @@
+#Submitted by Keren Or Hadad 208025205
+
+import numpy as np
+#1
+x = np.arange(21)
+print("my vector:")
+print(x)
+print("After changing the sign of the numbers in the range from 9 to 15:")
+x[(x >= 9) & (x <= 15)] *= -1
+
+print(x)
+
+"""
+output:
+my vector:
+[ 0  1  2  3  4  5  6  7  8  9 10 11 12 13 14 15 16 17 18 19 20]
+After changing the sign of the numbers in the range from 9 to 15:
+[  0   1   2   3   4   5   6   7   8  -9 -10 -11 -12 -13 -14 -15  16  17
+  18  19  20]
+"""
+
+#2
+
+v = np.linspace(10, 49, 5)
+print("Length 10 with values evenly distributed between 5 and 50:")
+print(v)
+
+"""
+output:
+Length 10 with values evenly distributed between 5 and 50:
+[10.   19.75 29.5  39.25 49.  ]
+"""
+
+#3
+
+m= np.arange(10,22).reshape((3, 4))
+print("Original matrix:")
+print(m)
+print("Number of rows and columns of the said matrix:")
+print(m.shape)

```

Comment:
This question should definitely be a function, it should work for any given matrix.